pages/api/dev/get-info.ts

```diff
@@ -0,0 +1,28 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default function route(req: NextApiRequest, res: NextApiResponse): Promise<void> {
+	return new Promise((resolve, reject) => {
+		if (req.body.user.length < 5) {
+			resolve();
+			res.status(404).send({ "message": "user does not exist" });
+		}
+		const data = {
+			followers: "50",
+			following: "70",
+			scannedProducts: "30",
+			userID: "mockUserId",
+			description: "Hi! you are reading my description 📓! I like using GoodBuy 😀. I care about the environment 🌳.",
```

Comment:
😁

Reply:
Those will disappear once getting user info will be possible through a real endpoint, don't worry

Reply:
No, no worries, I found that line indeed funny. 😁